src/organisms/Breadcrumbs.jsx

```diff
@@ -63,36 +65,57 @@ const Breadcrumbs = () => {
     }
   } else if (pathArray[0] === "project-setup") {
     appendToParentList("project setup", "/project-setup");
-    if (pathArray.length === 3) {
-      appendToParentList("Confirmation", "/project-setup/confirmation");
+    if (pathArray[1] !== undefined && pathArray[2] !== undefined) {
+      appendToParentList(pathArray[1], `/project-setup/${pathArray[1]}`);
+    }
+    //Adding this one since HCD is working on adding an extra page after selecting components
+    if (pathArray[2] !== undefined && pathArray[3] !== undefined) {
+      appendToParentList(
+        pathArray[2],
+        `/project-setup/${pathArray[1]}/${pathArray[2]}`
+      );
     }
   } else if (pathArray[0] === "projects") {
     appendToParentList("projects", "/projects");
     let title = "";
     if (state.project !== undefined && state.project.title !== undefined) {
       title = state.project.title;
     }
-
     if (pathArray.length === 2) {
       setPageName(title);
-    } else if (pathArray.length === 3) {
-      //potential pages they are on "Add Team Member", "Project Team", "Project Settings", "System Security Plan", "Manage Components", "System Controls"
+    }
+    if (pathArray[1] !== undefined && pathArray[2] !== undefined) {
       appendToParentList(title, `/projects/${pathArray[1]}`);
     }
+    if (pathArray[2] !== undefined && pathArray[3] !== undefined) {
```

Comment:
Nitpik(non-blocking): This is an example of good place to make a supporting method.  While I can read the code and tell that you're checking for undefined in two specific parts of the pathArray, writing a method that does this, while only used once, gives you two benefits 1) a method name that gives you more context than `pathArray != undefined` and 2) something specific that you can write a very easy unit test for so that the next dev that works on this component doesn't inadvertently break your logic.  In the consulting world, for a single dev to spend two years on a project is a long time so expect a bunch of people from potentially different companies to touch your code after you.